src/devices/logitech_gpro.c

```diff
@@ -0,0 +1,36 @@
+#include "../device.h"
+#include "../utility.h"
+#include "logitech.h"
+
+#include <string.h>
+
+static struct device device_gpro;
+
+static const uint16_t PRODUCT_ID = 0x0aa7; // PRO X: 0x0aaa
+
+static int gpro_send_sidetone(hid_device* device_handle, uint8_t num);
+
+void gpro_init(struct device** device)
+{
+    device_gpro.idVendor = VENDOR_LOGITECH;
+    device_gpro.idProductsSupported = &PRODUCT_ID;
+    device_gpro.numIdProducts = 1;
+
+    strncpy(device_gpro.device_name, "Logitech G PRO", sizeof(device_gpro.device_name));
+
+    device_gpro.capabilities = CAP_SIDETONE;
+    device_gpro.send_sidetone = &gpro_send_sidetone;
+
+    *device = &device_gpro;
+}
+
+static int gpro_send_sidetone(hid_device* device_handle, uint8_t num)
+{
+    // num = map(num, 0, 128, 0, 100);
+    if (num > 0x64)
+        num = 0x64;
```

Comment:
I'm not completely sure which solution is better.
The wiki example uses the `map` function, but the implementation in `logitech_g633_g933_935.c` uses this capping approach.

Reply:
Map is to be preferred. Because the user may supply a value from 0 - 128, and the cap simply ignores values after 100, which is confusing for the user.

Reply:
Ok, I will change it to map.